### PR TITLE
[Bug] Poison pill comparison was failing

### PR DESCRIPTION
**Pull request by scottf**

### CODE REVIEW

File: src/main/java/io/nats/client/impl/MessageQueue.java [hunk 1]
```diff
@@ -81,7 +85,7 @@ class MessageQueue {
         this.offerTimeoutMillis = Math.max(1, requestCleanupInterval.toMillis() * 95 / 100);
 
         // The poisonPill is used to stop poll and accumulate when the queue is stopped
-        this.poisonPill = new NatsMessage("_poison", null, EMPTY_BODY);
+//        this.poisonPill = new NatsMessage(POISON, null, EMPTY_BODY);
```
Comment (MauriceVanVeen) [inline src/main/java/io/nats/client/impl/MessageQueue.java:88]: Could these be removed now?

**Review by MauriceVanVeen — APPROVED** [review]
Small comment, but LGTM!

File: src/main/java/io/nats/client/impl/MessageQueue.java (hunk 1 above)
Comment (scottf) [reply src/main/java/io/nats/client/impl/MessageQueue.java]: removed

**Review by MauriceVanVeen — APPROVED** [review]
LGTM
Like the decrease of indents in `connectImpl`, etc. :+1:

**Comment by scottf** [comment]
@MauriceVanVeen can you recheck this change: https://github.com/nats-io/nats.java/pull/1162/files#diff-bbcf614ee7ee4ccd15b22d1f95fa40e4e642098386b6d1d14127dab5968d89feL377

**Pull request merged**